package.json

```diff
@@ -41,34 +41,18 @@
     "babel-loader": "^7.0.0",
     "babel-polyfill": "^6.23.0",
     "babel-preset-es2015": "^6.24.1",
-    "chai": "^3.5.0",
```

Comment:
Can you please update the `package-lock.json`? It can as well be done by whoever lands this though.